src/generate.ts

```diff
@@ -80,6 +80,25 @@ export const generate = (datadir: string) => {
     )
 }
 
+const optimism = ['optimism', 'optimism-goerli', 'optimism-sepolia']
+const base = ['base', 'base-goerli', 'base-sepolia']
+const pgn = ['pgn', 'pgn-sepolia']
+
+/**
+ * TODO: work out a better way to key these, perhaps by ID because
+ * the fallback to pgn isn't great for other Superchain networks
+ */
+const getTokenBridgeKey = (chain: string) => {
+  if (optimism.includes(chain)) {
+    return 'optimismBridgeAddress'
+  }
+  if (base.includes(chain)) {
+    return 'baseBridgeAddress'
+  }
+
+  return 'pgnBridgeAddress'
```

Comment:
```suggestion
  if (pgn.includes(chain)) {
  return 'pgnBridgeAddress'
  }
  throw new Error('unknown network ' + chain)
```
Prefer throwing to having a fallback. If we need a fall back we should fall back to optimismBaseAddress